src/lib.rs

```diff
@@ -12,24 +13,70 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! The `wkt` crate provides conversions to and from [`WKT`](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry) primitive types.
+//! See the [`types`](crate::types) module for a list of available types.
+//!
+//! Conversions (using [`std::convert::From`] and [`std::convert::TryFrom`]) to and from [`geo_types`] primitives are enabled by default, but the feature is **optional**.
+//!
+//! Enable the `serde` feature if you need to deserialise data into custom structs containing `WKT` geometry fields.
+//!
+//! # Examples
+//!
+//! ```
+//! use wkt::Wkt;
+//! let point: Wkt<f64> = Wkt::from_str("POINT(10 20)").unwrap();
+//! ```
+//!
+//! ```
+//! // Convert to a geo_types primitive from a Wkt struct
+//! use std::convert::TryInto;
+//! use wkt::Wkt;
+//! use geo_types::Point;
+//! // point's items field contains a wkt::Geometry enum member
```

Comment:
Yeah... items is weird. I guess the idea is that you can express multiple geometries without explicitly putting them into a GeometryCollection? 

I tried to track down the corresponding section in the spec, but chickened out after a few minutes.

http://docs.opengeospatial.org/is/18-010r7/18-010r7.html

Reply:
Maybe better to not even mention items in the introductory example, and only show it in the case where there are multiple items? Otherwise is there *any* reason to even know about items?

Reply:
I guess without the geo-types feature, `items` is the only way to get the geometry =(

Reply:
It would be great to know whether anyone actually uses the multiple geometry functionality in the wild…